docs/source/concepts/project_information.rst

```diff
@@ -0,0 +1,95 @@
+
+.. _concepts_project_information:
+
+Project Information Columns
+****************
+
+Introduction
+============
+
+For each project that you run in Cleanlab Studio, Cleanlab stores a number of metadata columns that can be retrieved using the Python API. 
+Currently we store the following project information columns:
+
+- Action Taken
+- If a row previously had an issue
+- Top Labels and Probabilities (alternative suggested labels and how likely they are)
+- The originally assigned (given) label and how likely it is to be the true label
+- The predicted label and its likelihood (even when we don't suggest changing from the given label).
+- If the row was initially unlabeled/had no given label. 
+
+To get this information, use ``studio.download_cleanlab_columns()`` and add the argument -p or -a for project details or all columns including analysis columns respectively.
```

Comment:
wait I don't understand what this line means. What is the argument you're supposed to pass into the `download_cleanlab_columns` function when you use the python CLI to get the project info columns?